torchao/experimental/ops/mps/test/test_quantizer.py

```diff
@@ -11,13 +11,27 @@
 import sys
 
 import torch
-import torchao_mps_ops
 import unittest
 
 from parameterized import parameterized
 from torchao.experimental.quant_api import UIntxWeightOnlyLinearQuantizer
 from torchao.experimental.quant_api import _quantize
 
+libname = "libtorchao_ops_mps_linear_fp_act_xbit_weight_aten.dylib"
+libpath = os.path.join(sys.prefix, "torchao_mps/cmake-out/lib/", libname)
+
+try:
+    torch.ops.load_library(libpath)
+except:
+    print(f"Failed to load library {libpath}")
+    raise
```

Comment:
This might need addressing when using this internally. I think the best way to do this is to do
```
try:
    for n bit.... loop to check for ops
except:
    try:
        load lib:
        for n bit...
    catch:
        throw error
```

Reply:
done!